trace/trace.go

```diff
@@ -235,7 +229,12 @@ func (t *Trace) Attach(c context.Context) context.Context {
 }
 
 // SpanFromContext is used to create a child span
-// when the parent trace is in the context
+// when the parent trace is in the context.
+//
+// Stability
+//
+// SpanFromContext is considered experimental. Prefer
```

Comment:
`is provided for experimental compatibility with opentracing`.

Reply:
Wait also the docs are wholly wrong. `SpanFromContext` does not create child spans, it only returns the currently active span. Now I get why I was confused about this function.

Reply:
I added a whole OT compatibility note because this function behaves confusingly different from the OT one.